client/rpcserver/types.go

```diff
@@ -42,7 +45,7 @@ type getFeeResponse struct {
 
 // tradeResponse is used when responding to the trade route.
 type tradeResponse struct {
-	OrderID string `json:"orderid"`
+	OrderID string `json:"orderID"`
```

Comment:
I don't know what passes for idiomatic json, but I though we were using all lower.  Leaving that up to @buck54321

Reply:
I've been inconsistent again. This is fine.

Reply:
@buck54321 I don't have an opinion, but I would like them to be the same for the same values. An all lower-case rule would make that simpler however. Let me know